Makefile

```diff
@@ -72,12 +76,9 @@ docker-dev: docker-build
 docker-push: docker-build
 	docker push $(IMAGE_NAME):$(IMAGE_VERSION)
 
-clean:
-	rm -rf .tmp
-	rm -rf phase3/.tmp
-	rm -rf phase1/gce/.tmp
-	rm -rf phase1/azure/.tmp
-	rm -rf phase1/vsphere/.tmp
+clean: .config.json
```

Comment:
@pipejakob, @divyenpatel, i have removed the dependency of clean from `.config.json` now.
PTAL